.github/workflows/coverage.yaml

```diff
@@ -9,15 +9,28 @@ jobs:
     steps:
     - name: Checkout code from "master"
       uses: actions/checkout@v1
+
     - name: Setup Java environment
       uses: actions/setup-java@v1
       with:
         java-version: 14
+
+    - name: Install Python for Test Dependencies
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install Poetry for Test Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install poetry
```

Comment:
It would be preferable to pin the `poetry` version here as well, so we don't get random crashes during testing due to some changes on their side.

Reply:
How about `pip` too?

Reply:
Sure, why not :)

Reply:
Locked pip & poetry.